setup.py

```diff
@@ -0,0 +1,17 @@
+"""
+setup
+"""
+from setuptools import setup
+
+setup(
+    name='template_python',
+    description='Template Python used in strg.AGENTS',
+    author='strg.at',
+    packages=[
+        'template_python.template_python_package'
+    ],
+    namespace_packages=['template_python'],
+    author_email='support@strg.at',
+    license='LGPL',
```

Comment:
consider adding a LICENSE file in the root folder. Is this true for all python repos?

Reply:
not sure if it is same for all the repos.
The `setup.py` provides the structure which needs to be changed as an when required.
There already exists a LICENSE file in the root folder.

Reply:
license is removed